library/enqueue-scripts.php

```diff
@@ -15,7 +15,8 @@
 
 if ( ! function_exists( 'foundationpress_asset_path' ) ) :
 function foundationpress_asset_path( $filename ) {
-	$dir = end( explode ( '.' , $filename) );
+	$tmp = explode('.', $filename);
```

Comment:
@JPOak please push another commit with the following changes:

1. Let's use a more descriptive name than `$tmp`. Change it to `$filename_split`.
2. Put spaces after opening parentheses and before closing parentheses.

Reply:
Ok Ok!